### PR TITLE
added line in README

### PR DESCRIPTION
**Pull request by cyprientemateu**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                        sudo docker run -itd -p 81:80 test:latest
+                        sudo docker ps 
+                        sudo docker ps -a
+                    """
+                }
+            }
+        }
+    }
+
+    post {
```
Comment (tia12) [inline Jenkinsfile:30]: Where I we sending these notifications

File: creative/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+WORKDIR /usr/local/apache2/htdocs/
+
+# Copy your local HTML file to the default Apache web root directory.
+COPY .  .
+# Expose the HTTPS port (443), or you can use a different port if needed.
+EXPOSE 80
+
+# Start Apache in the foreground when the container runs.
+CMD ["httpd-foreground"]
```
Comment (tia12) [inline creative/Dockerfile:11]: It is a good to use entrypoint to start the daemon

**Review by tia12 — CHANGES_REQUESTED** [review]

File: creative/Dockerfile (hunk 2 above)
Comment (deixdvops) [inline creative/Dockerfile:11]: I agree with Mr. Tia, an entrypoint will be more appropriate here

**Review by tia12 — APPROVED** [review]
It looks good

**Review by MNgouhouo — APPROVED** [review]

**Pull request merged**